tests/unit/test_get_clade_list.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+import pytest
+from virus_clade_utils.get_clade_list import main
+
+
+@pytest.fixture
+def test_file_path() -> Path:
+    """
+    Return path to the unit test files.
+    """
+    test_file_path = Path(__file__).parents[1].joinpath("data")
+    return test_file_path
+
+
+@pytest.mark.parametrize(
+    "threshold, weeks, max_clades, expected_list",
```

Comment:
These test cases are based on the data in `test_metadata.tsv`

The first three entries in each tuple represent the threshold, threshold_weeks, and max clades parameters for the `get_clade_list` process, and the fourth entry is the expected output (_i.e._, the list of clades to model).

This the most important thing to review: is the expected output correct, based on our test data file?